### DISCUSSION TITLE
Why don't you open up more experience slots, we are eager to use copilot

### DISCUSSION THREAD

**Discussion by jsdegithub** [community/community#6746] [Copilot Conversations]
Why don't you open up more experience slots, we are eager to use copilot

**Comment by Jobrien0394** [comment]
It's very buggy, luckily I got to use it, however, it's far from perfect. Plus I think I was on the waiting list for at least 3 months before I got the email.

**Comment by wait-what** [comment]
I wonder if the selection process takes account activity into consideration, or if it's purely based on luck every time some slots appear. I have been on the wait list for months and I still don't have access.

I would absolute love to try copilot, I am super excited about it. It really sucks to be the only one in the friend circle without access.

**Reply by richardalmonte** [reply]
I think that the selection doesn't consider the account activity. A colleague of mine got access even though having and account almost empty. And I registered the day after the launch and haven't received the access yet.

**Reply by Urethramancer** [reply]
It seems very random, yes. I got access mere days after applying, but just long enough that I'd forgotten about it. Co-workers have applied weeks ago but not seen any responses.
